lib/components/card.dart

```diff
@@ -1,4 +1,6 @@
 import 'package:flutter/material.dart';
+import 'package:font_awesome_flutter/font_awesome_flutter.dart';
```

Comment:
You do not need a new library for one icon that already exists in the built in material library.

https://api.flutter.dev/flutter/material/Icons-class.html
https://material.io/resources/icons/?style=baseline

location_on is the icon you want from here